openapi/spec/resources/devices/auth.yaml

```diff
@@ -15,47 +15,110 @@ post:
   parameters:
     - name: X-Real-IP
       schema:
-        description: Device IP
+        description: Device's IP
         type: string
+        pattern: ^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$
+        example: 127.0.0.1
       in: header
   requestBody:
     content:
       application/json:
         schema:
           type: object
           properties:
-            uid:
-              description: Device UID
-              type: string
-            token:
-              description: Device token
+            info:
+              type: object
+              properties:
+                id:
+                  description: Device's OS name
+                  type: string
+                  example: example
+                pretty_name:
+                  description: Device's OS pretty name
+                  type: string
+                  example: linux
+                version:
+                  description: Device's OS version
+                  type: string
+                  example: latest
+                arch:
+                  description: Device's OS arch
+                  type: string
+                  # TODO: add more compatible arches.
+                  enum:
+                    - x86_64
+                    - amd64
+                    - arm64
```

Comment:
I'd not validate it, at this moment. We support anything which runs Go.

Reply:
Perfect!